payment/app.py

```diff
@@ -72,28 +72,31 @@ def pay():
 def watch(service_id):
     body = request.get_json()
     body['service_id'] = service_id
+    order_id = body.pop('order_id')
     if request.method == 'DELETE':
         watcher = Watcher.get(service_id)
         if watcher:
-            watcher.remove_address(body['wallet_address'])
+            watcher.remove_address(body['wallet_address'], order_id)
```

Comment:
why remove is for a single item and add/set is for multiple?

Reply:
Set is used at the beginning to create watchers for all the known accounts in one call
add is multiple because that's how it was when we got the code and i don't want to change the old code too much yet.

And remove should only ever be called for a single account at once